app/handlers/moderator.py

```diff
@@ -272,12 +273,23 @@ async def get_info_about_user(
     target_karma = await user_repo.get_karma(target, chat)
     if target_karma is None:
         target_karma = "пока не имеет кармы"
-    information = f"Данные на {target.mention_link} ({target_karma}):\n" + "\n".join(
-        info
-    )
+
+    if info:
+        title = f"Данные на {target.mention_link} ({target_karma}):"
+    else:
+        title = f"Данные на {target.mention_link} ({target_karma}) отсутствуют"
+    info.insert(0, title)
+
     try:
         await bot.send_message(
-            message.from_user.id, information, disable_web_page_preview=True
+            chat_id=message.from_user.id,
+            text="\n".join(info),
+            reply_parameters=ReplyParameters(
+                message_id=message.message_id,
+                chat_id=message.chat.id,
+                allow_sending_without_reply=True,
+            ),
+            disable_web_page_preview=True,
```

Comment:
I think we need to use [LinkPreviewOptions](https://core.telegram.org/bots/api#linkpreviewoptions) instead of `disable_web_page_preview`, because it's new Bot API 7.0 feature and `disable_web_page_preview` remains only for backward compatibility